src/main/java/ua/danit/final_project/services/WorkCommentServiceImpl.java

```diff
@@ -3,9 +3,7 @@
 import org.joda.time.DateTime;
 import org.springframework.beans.factory.annotation.Autowired;
 import org.springframework.stereotype.Service;
-import ua.danit.final_project.configuration.StaticCollection;
 import ua.danit.final_project.entities.ShiftComment;
-import ua.danit.final_project.entities.WorkShift;
 import ua.danit.final_project.repositories.ShiftCommentRepository;
 import ua.danit.final_project.repositories.WorkShiftRepository;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unused import 'ua.danit.final_project.repositories.WorkShiftRepository'. ([squid:UselessImportCheck](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3AUselessImportCheck))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=88&issues=AWWWGKp17khxiKmGhe5I&open=AWWWGKp17khxiKmGhe5I)

[ISSUE_KEY]: AWWWGKp17khxiKmGhe5I